lib/stat_tracker.rb

```diff
@@ -1,17 +1,35 @@
+require_relative 'games'
+require_relative 'teams'
+require_relative 'game_teams'
+
 class StatTracker
   attr_reader :game_path,
               :team_path,
-              :game_teams_path
+              :game_teams_path,
+              :game_teams,
+              :teams,
+              :games
 
   def initialize(locations)
     @game_path = locations[:games]
     @team_path = locations[:teams]
     @game_teams_path = locations[:game_teams]
+    @game_teams = GameTeams
+    @teams = Teams
+    @games = Games
+
+    create_databases
```

Comment:
Is the reason we call create_databases in the initialize method  to ensure that the necessary data is loaded and ready to use immediately upon creation?

Reply:
Yes it is! We want to make sure that form the moment we create our StatTracker, our 3 "database" classes are ready to go as well!

Reply:
Thanks for the clarification!